packages/edit-site/src/components/post-fields/index.js

```diff
@@ -344,6 +344,28 @@ function usePostFields( viewType ) {
 					return <time>{ getFormattedDate( item.date ) }</time>;
 				},
 			},
+			{
+				id: 'comment_status',
```

Comment:
We should disable sorting here, as it's not supported.

Reply:
Additionally I think filtering is not supported by the REST API too..

Reply:
Oh, thanks! I actually forgot all fields are part of DataViews, even if hidden. Fixed at fcb8cc4d50408ccf07baf43ceefb42d2fe4a0ad0